zstackctl/zstackctl/ctl.py

```
@@ -2985,7 +2985,7 @@ def check_msyql():
                 ctl.internal_run('mysql_process_list', '--check')
 
         def open_iptables_port(protocol, port_list):
-            distro = platform.freedesktop_os_release()['ID'].lower()
+            distro = get_distro_id()
```

Ruff (0.15.7)

[error] `get_distro_id` may be undefined, or defined from star imports

(F405)